prow/cluster/horologium_deployment.yaml

```diff
@@ -35,6 +35,7 @@ spec:
         image: gcr.io/k8s-prow/horologium:v20190125-2aca69d
         args:
         - --job-config-path=/etc/job-config
+        - --dry-run=false
```

Comment:
--confirm

Reply:
no, this will have to wait until you bump